### ISSUE TITLE
feature request: support package name

### ISSUE THREAD

**Issue by ivysrono** [rsteube/t4-launcher#8]
1.0 Seem support application name only.
However, you don't support `CJKV Unified Ideographs`.
For example, https://f-droid.org/packages/com.simplemobiletools.filemanager.pro/
In Chinese system it‘s name is `文件`, which can't be searched by T4-Launcher.
Support package name maybe easier than support CJKV.

**Comment by rsteube** [comment]
Yeah i think supporting chinese characters would exceed the aim of this app a bit. I am also not sure if 4 Buttons would cut it as it has quite a lot characters.
What might be an alternative is forcing the english locale for languages not using english characters when [loading the labels](https://github.com/rsteube/t4-launcher/blob/master/src/main/kotlin/com/github/rsteube/t4/LauncherAdapter.kt#L69).
So instead of `文件` the english application name would be used.
